components/koth-tile/koth-tile.tsx

```diff
@@ -1,51 +1,79 @@
 import {
   Avatar,
+  gql,
   React,
   useEffect,
-  useMemo,
-  usePollingQuery,
-  useQuery,
-  useRef,
-  useState,
+  useMutation,
+  useQuerySignal,
+  useSelector,
   useStyleSheet,
 } from "../../deps.ts";
 import { KOTHOrgUser } from "../../types/mod.ts";
-import { CROWN_ICON } from "../../shared/mod.ts";
-import { KOTH_ORG_CONFIG_QUERY, KOTH_USER_QUERY } from "./gql.ts";
+import {
+  CROWN_ICON,
+  hasPermission,
+  useOrgKothConfigQuery$,
+  useOrgUserWithRoles$,
+} from "../../shared/mod.ts";
+import { KOTH_USER_QUERY } from "./gql.ts";
 import ActivePowerups from "../active-powerups/active-powerups.tsx";
 import Tile from "../tile/tile.tsx";
 
 import styleSheet from "./koth-tile.scss.js";
 
+const DELETE_KOTH_MUTATION = gql`
+mutation {
+  orgConfigUpsert(input: { data: { kingOfTheHill: { userId: "" } } }) {
+    orgConfig {
+      data
+    }
+  }
+}
+`;
+
 export default function KothTile() {
   useStyleSheet(styleSheet);
+  const orgUserWithRoles$ = useOrgUserWithRoles$();
+  const { orgKothConfig$, reexecuteKothConfigQuery } = useOrgKothConfigQuery$();
+  const [_deleteKOTHResult, executeDeleteKothResult] = useMutation(DELETE_KOTH_MUTATION);
+  useEffect(() => {
+    const id = setInterval(() => {
+      reexecuteKothConfigQuery({ requestPolicy: "network-only" });
+    }, 10000);
 
-  const kothUserIdRef = useRef<string>(undefined!);
-  const [kothUserId, setKothUserId] = useState("");
-  const orgKothConfigResponse = usePollingQuery(10000, { query: KOTH_ORG_CONFIG_QUERY });
-  const orgKothConfig = orgKothConfigResponse?.data;
-
-  const [{ data: kothUser }] = useQuery({
-    query: KOTH_USER_QUERY,
-    variables: {
-      userId: kothUserId,
-    },
-    requestPolicy: "network-only",
-    context: useMemo(() => ({ additionalTypenames: ["Org", "OrgConfig"] }), []),
-  });
+    return () => clearInterval(id);
+  }, []);
 
-  kothUserIdRef.current = orgKothConfig?.org?.orgConfig?.data?.kingOfTheHill?.userId;
+  const kothUserId = useSelector(() =>
+    orgKothConfig$.value.get!()?.data?.org?.orgConfig?.data?.kingOfTheHill?.userId
```

Comment:
did we get anywhere with figuring out if we can get the object w/o using `.get`? in instances where we don't care about nested observables (most instances)

Reply:
it's a little confusing, when creating an observable w/ legend it will create a proxy for every node in the object tree which wraps the node with the observable functions. 
https://legendapp.com/open-source/state/fast/#proxy-to-path

Using `.get()` returns the actual value vs. the wrapping proxy and also lets legend know that the component needs to be rerendered whenever that node in the object tree is updated. 
https://legendapp.com/open-source/state/tracking/#get
https://legendapp.com/open-source/state/observable/#get

All of the examples in the legend-state docs use .get when accessing a value so from my understanding it seems like we should default to using that method anytime we need to access an underlying value. 

We could open also open an issue and see if we can get some more context.